Jcn 355 subtarea sumar configuracion para evitar cerrar conexiones mongo (#26)
janis-commerce/mongodb: README.md

### Events and closing connections

To close connections, it is optional to define the environment variable `CLOSE_MONGODB_CONNECTIONS` in `Serverless.js` file.
Connections will be closed according to `CLOSE_MONGODB_CONNECTIONS` environment variable value:

> | Value | Close connection |

Reviewer: Agregaria una última columna que por ejemplo, el primero, se podría aclarar que "es equivalente a que no existe y es el comportamiento por default".
Y en el ultimo que "cualquier otro valor diferente a los anteriores"

|-------|------------------|

> | `not exist` | :white_check_mark: |

Reviewer: ```suggestion
| `undefined` | :white_check_mark: |
```

> | `false` | :x: |

Reviewer: ```suggestion
| `'false'` | :x: |
```

> | `undefined` | :white_check_mark: | Is equivalent to nonexistent and is the default behavior |

Reviewer: ```suggestion
| `undefined` (no value set) | :white_check_mark: | **Default behavior**, if you not set any value or create de env variable |
```

> | `exists with any other value` | :white_check_mark: |

Reviewer: ```suggestion
| `1`, `'true'`, etc.. (any other value) | :white_check_mark: |
```

> | `'false'` | :x: |  |

Reviewer: ```suggestion
| `'false'` | :x: | The connections to Database will keep open |
```

> | `1`, `'true'`, etc... | :white_check_mark: | any other value different from the above |

Reviewer: ```suggestion
| `1`, `'true'`, etc (any value set) | :white_check_mark: | Any other value different from the above. :warning:  We recommen to use `true` or no set any value |
```
